src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.skia.cs

```diff
@@ -71,6 +74,35 @@ protected override Size ArrangeOverride(Size finalSize)
 			return base.ArrangeOverride(finalSize);
 		}
 
+		/// <summary>
+		/// Gets the line height of the TextBlock either 
+		/// based on the LineHeight property or the default 
+		/// font line height.
+		/// </summary>
+		/// <returns>Computed line height</returns>
+		internal float GetComputedLineHeight()
+		{
+			var lineHeight = LineHeight;
+			if (!lineHeight.IsNaN() && lineHeight > 0)
+			{
+				return (float)lineHeight;
+			}
+			else
+			{
+				if (_cachedFontLineHeight is null)
```

Comment:
Do we really need to cache the the line height? I think the font itself is cached, then accessing `LineHeight` should be fast. Anything I'm missing?

Reply:
True!